hack/verify-ioutil-deprecation.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#This script checks the source code for deprecated methods in the
+# ioutil package that need to be replaced
+
+# Usage: `hack/verify-ioutil-deprecation.sh`.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+cd "${KUBE_ROOT}"
+
+error_messages=""
+
+find_files() {
+  # shellcheck disable=SC2207
```

Comment:
this is a valid lint that should not be ignored. we have a mapfile like helper in the shell lib, or you can use a redirect + for loop.